### ISSUE TITLE
Monitoring architectural diagram updates

### ISSUE THREAD

**Issue by lcawl** [elastic/stack-docs#440]
The diagrams in https://www.elastic.co/guide/en/elastic-stack-overview/master/how-monitoring-works.html do not reflect the collection of Logstash and Beats monitoring data via Metricbeat. They should perhaps also be updated to reflect the use of Filebeat to send log information to the Monitoring UI in Kibana.

Related to https://github.com/elastic/stack-docs/pull/404, which commented out the out-dated diagrams.

**Comment by lcawl** [comment]
Ideally our diagrams will align well with the style used in the Beats documentation (e.g. https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html) cc @dedemorton

**Comment by dedemorton** [comment]
Good idea, but it's worth noting that the diagram in the Beats documentation uses some older images. Updating it is kind of low on my priority list at the moment, TBH.
